db/db_wal_test.cc

```diff
@@ -756,7 +756,8 @@ class RecoveryTestHelper {
   // Manuall corrupt the specified WAL
   static void CorruptWAL(DBWALTest* test, const Options& options,
                          const double off, const double len,
-                         const int wal_file_id, const bool trunc = false) {
+                         const int wal_file_id, MockEnv* mem_env,
+                         const bool trunc = false) {
```

Comment:
Let me take look at the actual tests.
In the worst case, at least what we can do is to use MemEnv in those tests on Linux platform, which is covered by our internal CI, and leave the Mac platform in place.